ccswbs/gus
# Update UofG-styles stylesheet link to use JSDelivr CDN

# Summary of changes
Use CDN for UofG-styles instead of loading from files.web.uoguelph.ca

## Frontend
- Update link in gatsby-ssr.js

[x] My changes are accessible (at minimum WCAG 2.0 Level AA)
[x] My changes are responsive and appear as expected on mobile and desktop views.
[x] After merging, I will update the [Content Hub documentation](https://uoguelphca.sharepoint.com/sites/UniversityContentHubInformationGroup) and ensure the Content Hub trainer understands how my changes will affect users.

## Backend
N/A

# Test Plan

1. Visit the preview site https://deploy-preview-263--ugconthub.netlify.app/
2. Ensure that all styles are still the same as live
